sys/fido2/ctap/Kconfig

```
@@ -4,13 +4,31 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
-menuconfig KCONFIG_USEMODULE_FIDO2_CTAP
+menuconfig MODULE_FIDO2_CTAP
     bool "FIDO2 CTAP"
-    depends on USEMODULE_FIDO2_CTAP
+    depends on HAS_PERIPH_GPIO
```

Seems to be missing flashpage?
```suggestion
    depends on HAS_PERIPH_GPIO
    depends on HAS_PERIPH_FLASHPAGE
    select MODULE_PERIPH_FLASHPAGE
```

---

I am select MODULE_MTD_FLASHPAGE which depends and selects MODULE_PERIPH_FLASHPAGE. Should I add the depends nevertheless ?

---

I'd add the dependency to `HAS_PERIPH_FLASHPAGE` then at least.

---

Done :)